prime-router/src/test/kotlin/fhirengine/translation/hl7/FhirTransformerTests.kt

```diff
@@ -332,42 +352,222 @@ class FhirTransformerTests {
 
     @Test
     fun `test set bundle property failures`() {
+        val (transformer, logger) = setupFhirTransformer(FhirTransformSchema())
+
         val bundle = Bundle()
         bundle.id = "abc123"
-        val patient = Patient()
-        patient.id = "def456"
-        bundle.addEntry().resource = patient
-        val transformer = FhirTransformer(FhirTransformSchema())
 
-        // Incompatible value types
+        // Can't currently create entry on the fly
         assertThat {
             transformer.setBundleProperty(
-                "Bundle.entry.resource.ofType(Patient).name.text", CodeableConcept(),
+                "Bundle.entry.resource.ofType(DiagnosticReport).status", CodeType("final"),
                 CustomContext(bundle, bundle), bundle, bundle
             )
-        }.isFailure()
+        }.isFailure().hasMessage("Can't add missing entry.")
+
+        val patient = Patient()
+        patient.id = "def456"
+        bundle.addEntry().resource = patient
+
+        // Can't currently create new resources on the fly
         assertThat {
             transformer.setBundleProperty(
-                "Bundle.entry.resource.ofType(Patient).active", StringType("nonBoolean"),
+                "Bundle.entry.resource.ofType(DiagnosticReport).status", CodeType("final"),
                 CustomContext(bundle, bundle), bundle, bundle
             )
         }.isFailure()
 
-        // Can't currently create new resources on the fly
+        // Improper extension format
+        transformer.setBundleProperty(
+            "Bundle.entry.resource.ofType(Patient).extension(regexNonMatch).value[x]", IdType("newId"),
+            CustomContext(bundle, bundle), bundle, bundle
+        )
+        verify(exactly = 1) { logger.warn(match<String> { it.contains("Could not find property") }) }
```

Comment:
Brittle tests. Remove all the tests for the specific message in the log. This also makes it sound like it should be an error.

Reply:
Changed to check for an error without checking for specific message.